src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -215,7 +215,7 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
       }
     });
 
-    const items = this.props.items ?? allItems;
```

Comment:
the only time `this.props.filterItems` is applied is in the `allItems` getter, which doesn't get applied here if `this.props.items` is set.